src/app/loans/loans-account-stepper/loans-account-terms-step/loans-account-terms-step.component.html

```
@@ -377,6 +377,19 @@ <h4 class="mat-h4 flex-98">{{ 'labels.inputs.Loan Schedule' | translate }}</h4>
             <mat-option [value]="transactionProcessingStrategy.code">
               {{ transactionProcessingStrategy.name | translateKey: 'catalogs' }}
             </mat-option>
+          {{ 'labels.inputs.Allow full term for each tranche' | translate }}
+        </mat-checkbox>
```

HTMLHint (1.9.2)

[error] Tag must be paired, no start tag: [ </mat-checkbox> ]

(tag-pair)